### PR TITLE
Update feature_request.md

### PR DESCRIPTION
**Pull request by teolemon**
Better way to quantify impact

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -18,3 +18,9 @@ A clear and concise description of any alternative solutions or features you've
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+**Number of products impacted**
+If relevant, number (or estimate) of products impacted by the bug or feature at the time of creation of the issue
```
Comment (hangy) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This template is only for features, though.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -18,3 +18,9 @@ A clear and concise description of any alternative solutions or features you've
...
+
+**Timeloss per product**
```
Comment (hangy) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Shouldn't this be the other way around? Timeloss is a bit negative wording for a feature request, isn't it?

Comment (teolemon) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
**Time per product**
```

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (teolemon) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
If relevant, number (or estimate) of products impacted by the feature at the time of creation of the issue
```

**Pull request merged**